Architecture.md

```diff
@@ -16,7 +16,11 @@
 
 #### CLI (Command-Line Interface)
 
-***TODO: [CLIENT] fill for the cli***
+The CLI tool, or Command Line Interface, which is the main way for a User to create and deploy Projects, has been developed in [Go](https://go.dev/). It was chosen for its ease of use, its cross-platform compatibility and its performance. Moreover, the Go language is well suited for CLI development, as it is a compiled language, which allows for a single binary to be distributed, and it is used in many other CLI tools, such as Docker, Kubernetes, Terraform, and more.
```

Comment:
> which allows for a single binary to be distributed
no ? See https://go.dev/doc/install/source#environment GOOS and GOARCH

Reply:
Thanks for the info, this has been removed.